extensions/vscode/src/test/unit-test-utils/factories.ts

```diff
@@ -70,7 +70,7 @@ export const contentRecordFactory = Factory.define<ContentRecord>(
     saveName: `Report ${sequence}`,
     createdAt: new Date().toISOString(),
     localId: "",
```

Comment:
Similar note about the removal of `localId`

Reply:
thanks again.